src/instances/RaidInstance.ts

```diff
@@ -884,7 +884,7 @@ export class RaidInstance {
         if (!vc) return;
 
         if (!this._oldVcPerms) {
-            vc.setPosition(0).then();
+            vc.setPosition(this._raidSection.otherMajorConfig.afkCheckProperties.defaultPosition).then();
```

Comment:
For servers that Tooga joined before this patch (which is essentially every server that Tooga is in), if this isn't configured before an AFK check is started, `...afkCheckProperties.defaultPosition` will return `undefined`. This is because the database document has already been established before this field was added. So, will this cause a problem?

A simple fix is to just do
```
vc.setPosition(this._raidSection.otherMajorConfig.afkCheckProperties.defaultPosition ?? 0).then();
```